MediaBrowser.sln

```diff
@@ -525,6 +527,26 @@ Global
 		{9BA471D2-6DB9-4DBF-B3A0-9FB3171F94A6}.Release|x64.Build.0 = Release|Any CPU
 		{9BA471D2-6DB9-4DBF-B3A0-9FB3171F94A6}.Release|x86.ActiveCfg = Release|Any CPU
 		{9BA471D2-6DB9-4DBF-B3A0-9FB3171F94A6}.Release|x86.Build.0 = Release|Any CPU
+		{34905019-3332-43D2-8962-3217B69D81FF}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Need to check that all of these are for EmbyServer.Test, and not the configurations removed by @nvllsvm earlier.

Reply:
Confirmed that these are all for EmbyServer.Test